oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -562,7 +562,7 @@ private static String filenameToFamily(String name) {
         List<String> systemctl = ExecutingCommand.runNative("systemctl list-unit-files");
         for (String str : systemctl) {
             String[] split = ParseUtil.whitespaces.split(str);
-            if (split.length == 2 && split[0].endsWith(".service") && "enabled".equals(split[1])) {
+            if ((split.length == 2 || split.length == 3) && split[0].endsWith(".service") && "enabled".equals(split[1])) {
```

Comment:
Since we only use the first two columns, how about just `split.length >= 2`?  Then they can add even more columns later.